cdk/domino_cdk/provisioners/eks/eks_nodegroup.py

```diff
@@ -281,3 +260,31 @@ def _handle_user_data(
                 mime_user_data.add_part(ec2.MultipartBody.from_user_data(ud))
 
         return mime_user_data
+
+    def _launch_template(self, scope, name: str, ng: NodeGroup, **opts) -> ec2.LaunchTemplate:
+        root_device_name = "/dev/xvda"  # This only works for AL2
```

Comment:
root_device_name = "/dev/xvda"  # This only works for AL2 if not ng.ami_id else root_device_mapping(self.scope, ng.ami_id).name

I really don't like:
```
do-stuff
oh, wait, if some other stuff, then undo and redo stuff.
```
patterns

Reply:
Heh sure, updated.